io-engine/src/grpc/v1/snapshot_rebuild.rs

```diff
@@ -0,0 +1,49 @@
+use crate::grpc::GrpcResult;
+use io_engine_api::v1::snapshot_rebuild::{
+    CreateSnapshotRebuildRequest,
+    DestroySnapshotRebuildRequest,
+    ListSnapshotRebuildRequest,
+    ListSnapshotRebuildResponse,
+    SnapshotRebuild,
+    SnapshotRebuildRpc,
+};
+use tonic::{Request, Status};
+
+#[derive(Debug)]
+pub struct SnapshotRebuildService {
+    #[allow(unused)]
+    name: String,
+    #[allow(unused)]
+    replica_svc: super::replica::ReplicaService,
+}
+
+impl SnapshotRebuildService {
+    pub fn new(replica_svc: super::replica::ReplicaService) -> Self {
+        Self {
+            name: String::from("RebuildService"),
```

Comment:
Better to name it `SnapRebuildService` ?

Reply:
Done